### PR TITLE
Some css fixes

### PR DESCRIPTION
**Pull request by MonikaSharma06**
- [x] Remove border radius from all select input
- [x] Fixed alignment of fields in view mode
- [x] Made `height: auto` for codemirror editor
- [x] Changed opacity of configuration tree icon

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -462,4 +462,13 @@ span.tooltip {
   font-size: 0.9rem !important;
 }
 
+.select select {
+  border-radius: 0px;
```
Comment (praveen-garg) [inline src/styles.css:466]: On other controls like input, Do we have 0px or 2px?

File: src/app/components/core/configuration-manager/static-configuration/static-configuration.component.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
             class="readonly-editor" [disabled]="true">
           </ngx-codemirror>
         </ng-container>
-        <span *ngSwitchDefault class="help p-1 m-0">{{configuration.value ? configuration.value : "None"}}</span>
```
Comment (praveen-garg) [inline src/app/components/core/configuration-manager/static-configuration/static-configuration.component.html:39]: What was the issue with p-1 m-0? Just want to make sure, We know why are we removing this.

File: src/styles.css (hunk 1 above)
Comment (MonikaSharma06) [reply src/styles.css:466]: It is 2px and 4px on different pages. Should I change and make all of these consistent?

File: src/app/components/core/configuration-manager/static-configuration/static-configuration.component.html (hunk 2 above)
Comment (MonikaSharma06) [reply src/app/components/core/configuration-manager/static-configuration/static-configuration.component.html:39]: This was only for one type of control and it was causing alignment issue on configuration page. I don't see any need of it so I removed.

File: src/styles.css (hunk 1 above)
Comment (praveen-garg) [reply src/styles.css:466]: Yes please; make consistent as 2px

**Review by praveen-garg — DISMISSED** [review]

**Review by mshariq-nerd — APPROVED** [review]

**Review by hemantgarg923 — APPROVED** [review]

**Pull request merged**